### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by stephendonner**
Tested this locally (!); no ad-hoc run on staging Jenkins yet, due to Selenium-version testing going on there.  As mentioned, I'm doing this (and others) as a valid proof that I'm getting back into automation :-)

Run output (one failure due to missing data - I didn't use the rerun plugin):

https://pastebin.mozilla.org/8863178

### CODE REVIEW

**Comment by stephendonner** [comment]
@m8ttyB @davehunt @rbillings ?

File: requirements.txt [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # This pulls in all the libraries needed to run Selenium tests
-# on Mozilla WebQA projects
+# on Mozilla Web QA projects
```
Comment (davehunt) [inline requirements.txt]: Remove the comment, we've been doing this in other repos as part of the cleanup.

File: requirements.txt [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-execnet==1.0.9
-certifi==0.0.8
-chardet==1.0.1
-py==1.4.26
-pytest==2.7.0
+PyYAML==3.11
+execnet==1.4.1
+certifi==2016.2.28
+chardet==2.3.0
+py==1.4.31
```
Comment (davehunt) [inline requirements.txt]: Remove all downstream dependencies. I believe PyYAML, execnet, certifi, chardet, and py are all downstream dependecies. We let pytest determine its dependencies now.

File: requirements.txt [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+pytest==2.7.3
```
Comment (davehunt) [inline requirements.txt]: Latest is 2.9.0

**Comment by stephendonner** [comment]
(Ran out of time tonight; I'll fix up the rebase + squash tomorrow, sorry!)

**Comment by stephendonner** [comment]
Ad-hoc on prod: http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/socorro.adhoc/82/console
Ad-hoc on staging (w/known test failure): http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/socorro.adhoc/83/

Also tested locally - and rebased; I hope that's everything!

File: requirements.txt [hunk 4]
```diff
@@ -1,13 +1,6 @@
-# This pulls in all the libraries needed to run Selenium tests
-# on Mozilla WebQA projects
-
-PyYAML==3.10
-execnet==1.0.9
-certifi==0.0.8
-chardet==1.0.1
-py==1.4.26
-pytest==2.7.0
+browserid
```
Comment (davehunt) [inline requirements.txt]: I've been commenting on other patches that this should be `bidpom`, but it might not be simply a case of changing the package name. For this reason I think updating this package should be dealt with in a separate patch.

**Comment by davehunt** [comment]
r+

**Comment by rbillings** [comment]
Tests pass locally, changes look good. R+
